### ISSUE TITLE
Configure breaks looking for Win32 make file

### ISSUE THREAD

**Issue by Hypexed** [threader/e-uae-ppc-jit-nix#1]
Hello.

Downloaded and found some issues configuring. At first, scripts lacked execute bit, which I'm not used too. I tracked them down and set it.

But it broke at this point:
`config.status: error: cannot find input file: 'src/od-win32/Makefile.in'`

Cross compiling on x86-64 Linux:
`./configure --host=ppc-amigaos
`

**Comment by threader** [comment]
Hi, I noticed some issues when I finally a few moons ago got my powerbook updated to the latest debian ports, the last official debian release had something break picasso 96 mode in UAE so I originally threw in the towel there some years ago, but with the latest debian ports it worked again, turned out to be a kernel issue, I thought it was some library. I was hoping to fix things quickly for newer gcc's too but I'm a bit delayed, i found some issues with compiling with gcc-9 ( or as it 10) . 
I'm interested to see how it works for the amigaos target, I took some changes from p-uae to the amiga code, though that should "just work™". 

Thank you for the comment, I'll get it fixed once I've figured out the current kernel issue i have driving me crazy. (or i need a break from that)

**Comment by Hypexed** [comment]
Hi again. Regarding P96, according to Helge on Facebook, it only works in the SDL builds. This would explain why I could never get never get RTG working with my RTG config. Don't know why SDL is needed for RTG. But be good to merge what is needed for RTG in this build so we can have fast JIT and RTG. :-)

**Comment by threader** [comment]
Sorry, didn't spot you're message. 
I think that is correct behavior, some sdl calls to opengl etc, but don't quote me on that, it's been so long since I last had a proper look at this. and probably best i stopped there before i broke something with PUAE code.

**Comment by threader** [comment]
I set out to fix this issue the other day only to discover that the autoconf/make scripts were broken, so instead of fixing the issue then i ended up just making auto tools happy again. But I will try to make this work again for newer GCC's in the time ahead, and at least fix the low hanging fruit, but I'm currently facing some unexpected issues, I'm still unsure if it's my running kernel etc. as I've got no recollection on how I set up Yaboot and which kernel was which, but it dies on P96. So slowly but surely. I'll fix and close this issue during the day.
